src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
line no: 2 - Method name 'stringEqualityCheckNoncompliant' is not clear and should be more descriptive